tests_new/test_covariance_cluster_counts.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/python3
+import os
+import numpy as np
+import pyccl as ccl
+from tjpcov_new.covariance_clusters import CovarianceClusterCounts
+from tjpcov_new.covariance_io import CovarianceIO
+
+# INPUT
+# CCL and sacc input:
+os.makedirs("tests/tmp/", exist_ok=True)
+cosmo_filename = "tests/data/cosmo_desy1.yaml"
+cosmo = ccl.Cosmology.read_yaml(cosmo_filename)
+
+# SETUP
+input_yml = "examples/clusters/tjpcov_conf_minimal_clusters.yaml"
+cluster_count_cov = CovarianceClusterCounts(input_yml)
+
+
+def get_config():
+    return CovarianceIO(input_yml).config
+
+
```

Comment:
I think you should add tests for all the functions that will survive in TJPCov once you move code to CCL (so `get_covariance_block`'s and any other cluster specific thing that you have implemented)

Reply:
Added test for get_covariance_block (through `get_covariance_cluster_counts`)